### PR TITLE
SQ-60/Create location onboarding

### PR DESCRIPTION
**Pull request by rock3r**
This PR creates the basic framework for onboarding and the MVP for the location onboarding.

location_onboarding

TODO:
 * Use correct logo (we don't have it yet, waiting for @DigitalPencils)
 * Extract permissions/BT logic to a collaborator we can use elsewhere (e.g., `HomeActivity`) to check and re-request permissions/BT as needed
 * Create `ProximityFeature` that enables the proximity stuff (including the onboarding!) IIF:
    * The device has a Bluetooth radio
    * The device can request location updates (is there a way to check this? not sure)
    * The remoteconfig says we can enable it
    * The user has opted in to the location services during the onboarding or from the settings

### CODE REVIEW

File: app/src/main/res/layout/activity_location_onboarding.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  <ImageView
+    style="@style/Onboarding.Logo"
+    android:layout_width="wrap_content"
+    android:layout_height="@dimen/onboarding_logo_height"
+    android:layout_marginStart="@dimen/onboarding_content_margin_horizontal"
+    android:layout_marginTop="@dimen/onboarding_logo_margin_top"
+    android:layout_marginEnd="@dimen/onboarding_content_margin_horizontal"
+    android:contentDescription="@null"
+    android:src="@drawable/google_primary" />
```
Comment (quiro91) [inline app/src/main/res/layout/activity_location_onboarding.xml]: since we already had issues with vector drawables can we just use `app:srcCompat` ?

**Review by quiro91 — COMMENTED** [review]
I love it, just one small comment to address and it's good for me

File: app/src/main/res/layout/activity_location_onboarding.xml (hunk 1 above)
Comment (rock3r) [reply app/src/main/res/layout/activity_location_onboarding.xml]: Ooops totally forgot :P

File: app/src/main/java/net/squanchy/navigation/Navigator.java [hunk 2]
```diff
@@ -26,51 +27,48 @@
 
 public class Navigator {
 
-    private final Context context;
+    private static final int NO_FLAGS = 0;
```
Comment (quiro91) [inline app/src/main/java/net/squanchy/navigation/Navigator.java:30]: shouldn't there be an empty line between `static final` fields and other fields?

**Review by quiro91 — APPROVED** [review]
💯

File: app/src/main/java/net/squanchy/navigation/Navigator.java (hunk 2 above)
Comment (rock3r) [reply app/src/main/java/net/squanchy/navigation/Navigator.java:30]: ...yes there should. Will fix in next PR, takes too much to do a CI build for this XD

**Review by tiwiz — APPROVED** [review]

**Pull request merged**